driver-core/src/test/java/com/datastax/driver/core/ExportAsStringTest2x.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (C) 2012-2017 DataStax Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.driver.core;
+
+import com.datastax.driver.core.schemabuilder.SchemaBuilder;
+import com.datastax.driver.core.utils.CassandraVersion;
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import org.testng.SkipException;
+import org.testng.annotations.Test;
+
+import static com.datastax.driver.core.ExportAsStringTest.getExpectedCqlString;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@CassandraVersion("2.0")
+public class ExportAsStringTest2x extends CCMTestsSupport {
+
+    /**
+     * A version of {@link ExportAsStringTest} for C* 2.0 and 2.1 clusters.
+     */
+    @Test(groups = "short")
+    public void create_schema_and_ensure_exported_cql_is_expected() {
```

Comment:
This is a duplicate of `ExportAsStringTest`, but for 2.0 and 2.1 only.  Separated them out before `ExportAsStringTest` requires UDFs to be enabled, but i just realized we already have selective logic to remove that configuration for older versions..will unify.